context/src/main/java/io/opentelemetry/context/ThreadLocalContextStorage.java

```diff
@@ -15,9 +17,8 @@
 
   private static final ThreadLocal<Context> THREAD_LOCAL_STORAGE = new ThreadLocal<>();
 
-  static {
-    THREAD_LOCAL_STORAGE.set(Context.root());
```

Comment:
I removed this since it only sets the value on whichever thread happens to run the static constructor.  Every other thread would remain uninitialized and will return `null`.  Overriding `initialValue()` or calling the static `withInitialValue` method can provide a default value but the latter seemingly doesn't exist in Android and it's not really necessary since read access to the `ThreadLocal` is always through `current()` anyway.